src/Installers/Windows/AspNetCoreModule-Setup/IIS-Common/build/sign.props

```diff
@@ -0,0 +1,24 @@
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MicroBuildVersion>0.2.0</MicroBuildVersion>
+  </PropertyGroup>
+
+<!-- Item Group evaluated after TargetName and TargetExt are defined -->
+  <ItemGroup Condition=" '$(SignType)' != ''">
+    <FilesToSign Condition="$(TargetExt) != '.msi'" Include="$(FilesToSign);$(OutDir)\$(TargetName)$(TargetExt)">
+      <Authenticode>$(SigningIdentity)</Authenticode> 
```

Comment:
I couldn't find where SigningIdentity is set, but it should be "Microsoft400". This is not the default in MicroBuild signing. See https://github.com/aspnet/Universe/blob/6f2ef9d886146106b9f1d14527c05728859ad29b/eng/targets/Wix.Common.targets#L4-L18

Reply:
I went ahead and removed all signing files. Working on implementing signing now (like we do with the hosting bundle).